reports_download.py

```diff
@@ -53,17 +57,17 @@
 login_but.click()
 time.sleep(10)
 
-
-# To click OOPS banner.
-oops_banner = driver.find_element_by_css_selector("body > app-root > div > app-student-home > div > mat-card > div > div > app-student-home-card > mat-card")
-oops_banner.click()
+# To click DAA banner.
```

Comment:
Can we make it generic, i know it was oops but let's make it generic.

Reply:
Yeah, generic is great. We can leave ```url``` blank with a comment to add whichever eLab the report downloading is for.

Reply:
Sure that seems like a plan, will merge it once you make those changes.